specification/translation/Azure.AI.DocumentTranslation/routes.tsp

```diff
@@ -211,18 +211,17 @@ interface DocumentTranslationOperations {
     Returns the translation status for a specific document based on the request Id
     and document Id.
     """)
-  @route("/document/batches/{id}/documents/{documentId}")
+  @route("/document/batches/{translationId}/documents/{documentId}")
   @get
   getDocumentStatus is Azure.Core.Foundations.Operation<
     {
-      @doc("Format - uuid.  The batch id")
-      @clientName("translation_id", "python")
+      @doc("Format - string.  The translation operation id")
       @path
-      id: uuid;
+      translationId: string;
 
-      @doc("Format - uuid.  The document id")
+      @doc("Format - string.  The document id")
       @path
-      documentId: uuid;
+      documentId: string;
```

Comment:
why did this change from uuid?

Reply:
This change was requested by @schaabs => uuid to string for all ids